Library/Homebrew/test/cask/artifact/shared_examples/uninstall_zap.rb

```diff
@@ -61,6 +61,19 @@
     end
   end
 
+  context "using launchctl with regex" do
+    let(:cask) { Cask::CaskLoader.load(cask_path("with-#{artifact_dsl_key}-launchctl-wildcard")) }
+    let(:launchctl_list_cmd) { %w[/bin/launchctl list] }
+
+    it "searches running launchctl items" do
+      allow(fake_system_command).to receive(:run)
```

Comment:
```suggestion
      expect(fake_system_command).to receive(:run)
```

assuming you want this to fail if not.